superset-frontend/src/common/components/Tooltip.tsx

```diff
@@ -0,0 +1,21 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import { Tooltip } from 'src/common/components';
+
+export default Tooltip;
```

Comment:
What's the thought-process around this wrapper component? I saw that we have similar ones that wrap antd but with our styles applied on top. Should the styled tooltip css above be in this file?

Reply:
I wanted to leave it open in case we wanted to add custom styles at a later time. It might make sense to take the custom styles in `InfoTooltip` and just apply them to the general `Tooltip` component here though!